kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/ClusterData.java

```diff
@@ -111,6 +116,8 @@ static ClusterData fromJson(
 
     public abstract Builder setTopicConfigs(Relationship topicConfigs);
 
+    public abstract Builder setReassignment(Relationship reassignment);
```

Comment:
setPartitionReassignments(Relationship partitionReassignments);

Reply:
Modified all getters and setters for ClusterData and TopicData classes and tests.